tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

```diff
@@ -1,5 +1,19 @@
 # Azure SDK Tools Test Proxy
 
+**To users writing SDK tests:** This document is not intended to be a test preperation reference. You should instead refer
+to documentation in your specific language repository in order to configure recorded tests.
+
+#### Test documentation by language:
```

Comment:
We should add this to the `asset-sync` / `test-proxy` readme in sdk tools repo shouldn't we.